packages/core/lib/commands/install.js

```diff
@@ -1,35 +1,54 @@
+const { callbackify } = require("util");
+
 const command = {
   command: "install",
   description: "Install a package from the Ethereum Package Registry",
   builder: {},
   help: {
-    usage: "truffle install <package_name>[@<version>]",
+    usage: "truffle install <package-identifier> [--alias]",
     options: [
       {
-        option: "package_name",
-        description:
-          "Name of the package as listed in the Ethereum Package Registry. (required)"
+        option: "packageIdentifier",
+        description: `(required) Name of the package as listed in the Ethereum Package Registry. Accepted formats: packageName, packageName@version, ethpm URI, ipfs URI.`
       },
       {
-        option: "<@version>",
-        description:
-          "When specified, will install a specific version of the package, otherwise " +
-          "will install\n                    the latest version."
+        option: "--alias",
+        description: "An alternate name under which to install this package."
       }
     ]
   },
-  run: function(options, done) {
+  run: function (options, done) {
     const Config = require("@truffle/config");
-    const Package = require("../package");
+    const PackageV1 = require("@truffle/ethpm-v1");
+    const PackageV3 = require("@truffle/ethpm-v3");
 
-    if (options._ && options._.length > 0) options.packages = options._;
+    if (options._ && options._.length == 0) {
+      done(
+        new Error(
+          `Please provide a package identifier. Run 'truffle help install' for more information on valid package identifiers.`
+        )
+      );
+    }
+    if (options._ && options._.length > 1) {
+      done(
+        new Error(
+          `Multiple package identifiers detected. Only one package can be installed at a time.`
+        )
+      );
+    }
+    options.packageIdentifier = options._[0];
 
     const config = Config.detect(options);
-    Package.install(config)
-      .then(() => {
-        return done();
-      })
-      .catch(done);
+    let callbackFunction;
+
+    if (config.ethpm.version == "1") {
+      callbackFunction = callbackify(PackageV1.install);
```

Comment:
This should probably be named something like `install` or `installPackage` as it is really not a callback function.